scripts/complete_liquidity_provision.js

```diff
@@ -182,32 +186,43 @@ module.exports = async (callback) => {
       true
     )
 
-    if (!argv.verify) {
-      console.log(
-        "==> Sending the order placing transaction to gnosis-safe interface.\n    Attention: This transaction MUST be executed first!"
-      )
-    } else {
-      console.log("==> Order placing transaction")
-    }
     let nonce = argv.nonce
     if (nonce === undefined) {
       nonce = (await masterSafe.nonce()).toNumber()
     }
-    await signAndSend(masterSafe, orderTransaction, argv.network, nonce, argv.verify)
-
     if (!argv.verify) {
+      console.log(
+        "==> Sending the order placing transaction to gnosis-safe interface.\n    Attention: This transaction MUST be executed first!"
+      )
+      await signAndSend(masterSafe, orderTransaction, argv.network, nonce)
       console.log(
         "==> Sending the funds transferring transaction.\n    Attention: This transaction can only be executed after the one above!"
       )
-    } else {
-      console.log("==> Funds transferring transaction")
-    }
-    await signAndSend(masterSafe, bundledFundingTransaction, argv.network, nonce + 1, argv.verify)
-
-    if (!argv.verify) {
+      await signAndSend(masterSafe, bundledFundingTransaction, argv.network, nonce + 1)
       console.log(
         `To verify the transactions run the same script with --verify --nonce=${nonce} --brackets=${bracketAddresses.join()}`
       )
+    } else {
+      console.log("==> Order placing transaction")
+      if (await transactionExistsOnSafeServer(masterSafe, orderTransaction, argv.network, nonce)) {
+        console.log(
+          `The order placing transaction built from the verification parameters matches the transaction in the interface! You can sign the transaction with nonce ${nonce} here: ${interfaceLink}`
+        )
+      } else {
+        console.log(
+          "Error: The order placement transaction built from the verification parameters does not match any transaction in the interface!"
+        )
+      }
+      console.log("==> Funds transferring transaction")
+      if (await transactionExistsOnSafeServer(masterSafe, bundledFundingTransaction, argv.network, nonce + 1)) {
+        console.log(
+          `The funds transferring transaction built from the verification parameters matches the transaction in the interface! You can sign the transaction with nonce ${nonce} here: ${interfaceLink}`
+        )
+      } else {
+        console.log(
+          "Error: The funds transferring transaction built from the verification parameters does not match any transaction in the interface!"
+        )
```

Comment:
Although this might look like code duplication, notice that the error messages are different. Of course we could come up with a more generic message here. My point was more about the fact the we need to remove the term "reconstructed" transaction since the function can be used for other purposes (i.e. like just verifying if the same transaction already exists when it has not been reconstructed). 

I would be ok with the old implementation if these ideas are considered.